casa/OS/HostInfo.h

```diff
@@ -131,12 +131,22 @@ class HostInfo
     static ptrdiff_t swapFree();
     // </group>
 
+    // Introduce run-time memory limit re-specification capabilities
```

Comment:
missing documentation that this is only used when aipsrc is used too

also why is that so?